### PR TITLE
Suggest documentation on avatars

### PR DESCRIPTION
**Pull request by SMillerDev**
Recently I found that there was no docs on how to get the user avatar so I figured I'd add it.

### CODE REVIEW

**Review by MorrisJobke — APPROVED** [review]

**Comment by MorrisJobke** [comment]
cc @rullzer @tobiasKaminsky

**Review by rullzer — APPROVED** [review]

File: developer_manual/client_apis/OCS/ocs-api-overview.rst [hunk 1]
```diff
@@ -77,6 +77,24 @@ This request returns the available metadata of a user. Admin users can see the i
...
+-------------
+
+Since: 11.0.2, 12.0.0
+
+This request returns the available avatar of a user. The avatar is always a square so PIXEL_SIZE is both height and width. 
+The version parameter is optional, the last version will be returned if ommited.
+
+.. code::
+
+	GET /avatar/USERID/PIXEL_SIZE?v=VERSION
```
Comment (tobiasKaminsky) [inline developer_manual/client_apis/OCS/ocs-api-overview.rst:91]: I think we should add /index.php/ in front, as not all server have the rewrite to omit index.php.

**Comment by tobiasKaminsky** [comment]
Also we could add eTag support info.
Not sure if this is somewhere else mentioned in general, so we can only point to the concepts of eTags.

File: developer_manual/client_apis/OCS/ocs-api-overview.rst (hunk 1 above)
Comment (MorrisJobke) [reply developer_manual/client_apis/OCS/ocs-api-overview.rst:91]: Also this is not an OCS API URL, but a normal URL. So it needs another page in the documentation anyways

**Review by MorrisJobke — CHANGES_REQUESTED** [review]

File: developer_manual/client_apis/OCS/ocs-api-overview.rst (hunk 1 above)
Comment (SMillerDev) [reply developer_manual/client_apis/OCS/ocs-api-overview.rst:91]: I just put the data where I would expect it as a developer. Is there something I still need to do here?

**Comment by SMillerDev** [comment]
Do I need to create a new page? Can I have some guidance here, because I have no idea how to do that and link it all up nicely or what the standard is for writing a new page. Is there anyway this could be merged and extended from there?

**Comment by MorrisJobke** [comment]
I would say so. @ChristophWurst Any good idea where to put this and how to structure those non-OCS APIs?

**Comment by ChristophWurst** [comment]
> I would say so. @ChristophWurst Any good idea where to put this and how to structure those non-OCS APIs?

Someone in `developer_manual/client_apis`, I guess. We could just add a page for avatars there and if there's more pages like these that make it worth grouping, we can create a new directory for them. Does that make sense?

**Comment by SMillerDev** [comment]
> Someone in `developer_manual/client_apis`, I guess. We could just add a page for avatars there and if there's more pages like these that make it worth grouping, we can create a new directory for them. Does that make sense?

Could someone please push an empty page wherever they think it's good? I'll move my content there then.

**Comment by ChristophWurst** [comment]
> Could someone please push an empty page wherever they think it's good? I'll move my content there then.

Are you using github just from the web? can you not create a page there?

**Comment by SMillerDev** [comment]
> Are you using github just from the web? can you not create a page there?

For this repo, yes. And if I can I don't know how. Nor do I know what is required to set up a page, what info it needs, links from other pages etc.

**Pull request closed without merging**